src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -1501,11 +1503,12 @@ def create_or_update_applicationdefinition(cmd, resource_group_name,
     for name_value in authorizations:
         # split at the first ':', neither principalId nor roldeDefinitionId should have a ':'
         principalId, roleDefinitionId = name_value.split(':', 1)
-        applicationAuth = ApplicationProviderAuthorization(
+        applicationAuth = ApplicationAuthorization(
```

Comment:
Why do we need to change model name here? Is it because there is a breaking change in the Python SDK?

Reply:
Yes, it was renamed in python SDK.

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/7803/files#diff-16674354391b2fa53446470be08ef3159d1e7ed959862d1d7d7471aa7fb5cb1cL1367-L1373